hooks/pre_gen_project.py

```diff
@@ -0,0 +1,42 @@
+"""
```

Comment:
How about like this (I will provide you as a pseudo-code for easier to read). This can avoid nested if and I think it is more straightforward.

```py
-----------
Web Variant
-----------
if variant == web then
  update web_variant = y
  update api_variant = n
elif variant == api then
  update web_variant = n
  update api_variant = y
elif variant == both then
  update web_variant = y
  update api_variant = y

-----------
CSS Addon
-----------
if web_variant == y then
  if css_addon == bootstrap then
    update bootstrap_addon = y
    update tailwind_addon = n
  elif css_addon == tailwind then
    update bootstrap_addon = n
    update tailwind_addon = y
  elif css_addon = none then
    update bootstrap_addon = n
    update tailwind_addon = n
else then
  update bootstrap_addon = n
  update tailwind_addon = n
```

Is it possible to set the default value to private variable?  If yes we can reduce complexity of the code for example if we set the default value to every private variable to `No` then code will be like 

```py
-----------
Web Variant
-----------
if variant == web then
  update web_variant = y
elif variant == api then
  update api_variant = y
elif variant == both then
  update web_variant = y
  update api_variant = y

-----------
CSS Addon
-----------
if web_variant == y then
  if css_addon == bootstrap then
    update bootstrap_addon = y
  elif css_addon == tailwind then
    update tailwind_addon = y
```

Reply:
OMG thanks @gutakk ! I though the template format did not offer else if condition (after trying else if, elsif and elseif ><). 
I'll change that! ;-)